resources/fetchers/iam_fetcher.go

```diff
@@ -46,25 +46,36 @@ type IAMResource struct {
 
 func (f IAMFetcher) Fetch(ctx context.Context, cMetadata fetching.CycleMetadata) error {
```

Comment:
would you mind adding documentation of what is being fetched (pwd policy and IAM user)?

Reply:
Sure thing